streaming/src/main/scala/org/apache/carbondata/streaming/StreamHandoffRDD.scala

```diff
@@ -177,7 +177,8 @@ class StreamHandoffRDD[K, V](
       carbonTable,
       segmentProperties,
       CompactionType.STREAMING,
-      carbonTable.getTableName
+      carbonTable.getTableName,
+      null
```

Comment:
It is better to reject setting streaming property when creating partition table, I think query may have issue if we passing null when compacting the streaming table

Reply:
Ok, will add the restrictions in another pr